README.md

```diff
@@ -1,92 +1,150 @@
+
+### Meson installation
+
+Building with Meson 1.6.0 installation results in build issues
+[meson: error: unrecognized arguments:](https://github.com/strawberrymusicplayer/strawberry-msvc/issues/6).
+
+A temporary solution is to remove Meson via Windows Settings | `Apps or remove programs`.
+Then install Meson via Python PIP system wide. Start | `x64 Native Tools Command Prompt for VS 2022` and type:
+
+```
+pip install meson
+```
+
+## Optional: Prebuilt binary dependencies
+
+Prebuilt MSVC binaries can be optionally used to speed-up the build process. When this step is skipped, all libraries and dependencies are build from source.
+
+To use prebuilds, download the following `tar.xz` files from Github [strawberry-msvc-dependencies/releases](https://github.com/strawberrymusicplayer/strawberry-msvc-dependencies/releases):
+- For debug, extract `strawberry-msvc-x86_64-debug.tar.xz` to `C:\strawberry_msvc_x86_64_debug`
+- For release, extract `strawberry-msvc-x86_64-release.tar.xz` to `C:\strawberry_msvc_x86_64_release`
+
+## Build Strawberry from source including dependencies
+
+- Set one debug or release in `Path` environment variable:
+    - For debug: `C:\strawberry_msvc_x86_64_debug\bin`, or:
+    - For release: `C:\strawberry_msvc_x86_64_release\bin`
+- Start | `x64 Native Tools Command Prompt for VS 2022`:
+```
+cd C:\data\projects\strawberry\strawberry-msvc
+```
+
+For debug build:
+```
+.\build.bat
+```
+
+For release build:
+```
+.\build.bat release
+```
+
+Strawberry and Windows installer executables are generated in:
+- For debug: `C:\data\projects\strawberry\msvc_\build_debug\strawberry\build\`
+- For release: `C:\data\projects\strawberry\msvc_\build_release\strawberry\build\`
+
+## Debug Strawberry with Visual Studio
+
+Debug build:
+- Make sure path `C:\strawberry_msvc_x86_64_debug\bin` is set in environment variables
+- Start | Visual Studio 2022
+- Open a local folder `C:\data\projects\strawberry\msvc_\downloads\strawberry`
+- Toolbar Project | CMake Settings:
+    - Select Configurations `x64-Debug`
+    - Add `CMake command arguments`: `-DARCH=x86_64 -DICU_ROOT=c:\\strawberry_msvc_x86_64_debug`
+- Select in the toolbar `x64-Debug`
+- Toolbar: Build | Build All
+
+Release build:
+- Make sure path `C:\strawberry_msvc_x86_64_release\bin` is set in environment variables
+- Start | Visual Studio 2022
+- Open a local folder `C:\data\projects\strawberry\msvc_\downloads\strawberry`
+- Toolbar Project | CMake Settings:
+    - Create new `x64-Release`
+    - Select Configurations `x64-Release`
+    - Add `CMake command arguments`: `-DARCH=x86_64 -DICU_ROOT=c:\\strawberry_msvc_x86_64_release`
+    - Save the changes
+    - In the toolbar select `x64-Release`
+    - Project | Delete Cache and Reconfigure
+    - Uncheck: `ENABLE_WIN32_CONSOLE`
+- Toolbar: Build | Build All
+
+Set breakpoint and press F5 to step through the code.
```

Comment:
This is wrong, F5 is continue, F10 is step.

Reply:
Reworded in 34af329fe7723cf1eda80f8c60be094db344f154.